dumper-integration-tests/redshift-tests/src/test/java/com/google/integration/SvvColumnsTest.java

```diff
@@ -0,0 +1,82 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.integration;
+
+import static com.google.base.TestBase.assertListsEqual;
+import static com.google.base.TestConstants.EXPORTED_FILES_BASE_PATH;
+import static com.google.base.TestConstants.PASSWORD_DB;
+import static com.google.base.TestConstants.SQL_REQUESTS_BASE_PATH;
+import static com.google.base.TestConstants.URL_DB;
+import static com.google.base.TestConstants.USERNAME_DB;
+
+import com.google.common.collect.LinkedHashMultiset;
+import com.google.pojo.SvvColumnsRow;
+import com.google.sql.SqlUtil;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class SvvColumnsTest {
+
+  private static final String SQL_PATH = SQL_REQUESTS_BASE_PATH + "svv_columns.sql";
+  private static final String CSV_FILE_PATH = EXPORTED_FILES_BASE_PATH + "svv_columns.csv";
+  private static Connection connection;
+
+  @BeforeClass
+  public static void beforeClass() throws SQLException {
+    connection = DriverManager.getConnection(URL_DB, USERNAME_DB, PASSWORD_DB);
+  }
+
+  @Test
+  public void svvColumnsTest() throws SQLException, IOException, CsvValidationException {
+    LinkedHashMultiset<SvvColumnsRow> dbList = LinkedHashMultiset.create();
+    LinkedHashMultiset<SvvColumnsRow> csvList = LinkedHashMultiset.create();
+
+    try (PreparedStatement preparedStatement =
+        connection.prepareStatement(SqlUtil.getSql(SQL_PATH))) {
+      ResultSet rs = preparedStatement.executeQuery();
+
+      while (rs.next()) {
+        dbList.add(SvvColumnsRow.create(rs));
+      }
+    }
+
+    FileReader fileReader = new FileReader(CSV_FILE_PATH);
+    try (CSVReader reader = new CSVReaderBuilder(fileReader).withSkipLines(1).build()) {
+      String[] line;
+      while ((line = reader.readNext()) != null) {
+        SvvColumnsRow csvRow = SvvColumnsRow.create(line);
+        csvList.add(csvRow);
+      }
+    }
+
+    LinkedHashMultiset<SvvColumnsRow> dbListCopy = LinkedHashMultiset.create(dbList);
+    csvList.forEach(dbList::remove);
+    dbListCopy.forEach(csvList::remove);
+
+    assertListsEqual(dbList, csvList);
```

Comment:
This method doesn't really check lists equality, consider moving the above subtractions/disjunction logic in the helper method? Also testing frameworks might already have useful assertions (and informative error messages) for this case, e.g. the `Truth` framework has a `Truth.assertThat(Multiset a).constainsExactlyElementsIn(Multiset b)`

Reply:
I did a thorough research together with @haselbach and that custom implementation checks exactly what we need: 4 possible cases that may happen after reduction.
Using asserts from Guava is lacking 1 case out of possible 4, plus logging isn't sufficient for after-test-investigation.

Reply:
Which comparison case is missing from Truth assertions? Beside that my suggestion would still be to move the cross-removal of items to `TestBase.assertListsEqual` (unless this is specific to this test). As for the name, "list" would imply an order-aware comparison which is not the case, it might help future readers to name it `assertMultisetsEqual` or `assertCollectionsEqual`.

Reply:
Done